x-pack/plugins/index_lifecycle_management/public/sections/policy_table/components/policy_table/policy_table.js

```diff
@@ -0,0 +1,389 @@
+  };
+
+  render() {
+    const {
+      totalNumberOfPolicies,
+      policyFilterChanged,
+      filter,
+      policies,
+      intl,
+      policyListLoaded,
+    } = this.props;
+    const { selectedPoliciesMap } = this.state;
+    const numSelected = Object.keys(selectedPoliciesMap).length;
+    let content;
+    let tableContent;
+    if (totalNumberOfPolicies || !policyListLoaded) {
+      if (!policyListLoaded) {
+        tableContent = <EuiLoadingSpinner size="m" />;
+      } else if (policies.length > 0) {
+        tableContent = (
+          <EuiTable>
+            <EuiTableHeader>{this.buildHeader()}</EuiTableHeader>
+            <EuiTableBody>{this.buildRows()}</EuiTableBody>
+          </EuiTable>
+        );
+      } else {
+        tableContent = <NoMatch />;
+      }
+      content = (
+        <Fragment>
+          <EuiFlexGroup gutterSize="l" alignItems="center">
+            {numSelected > 0 ? (
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  data-test-subj="deletePolicyButton"
+                  color="danger"
+                  onClick={() => this.setState({ showDeleteConfirmation: true })}
+                >
+                  Delete {numSelected} polic
+                  {numSelected > 1 ? 'ies' : 'y'}
+                </EuiButton>
+              </EuiFlexItem>
+            ) : null}
+            <EuiFlexItem>
+              <EuiFieldSearch
+                fullWidth
+                value={filter}
+                onChange={event => {
+                  policyFilterChanged(event.target.value);
+                }}
+                data-test-subj="policyTableFilterInput"
+                placeholder={intl.formatMessage({
+                  id: 'xpack.indexLifecycleMgmt.policyTable.systempoliciesSearchInputPlaceholder',
+                  defaultMessage: 'Search',
+                })}
+                aria-label="Search policies"
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="m" />
+          {tableContent}
+        </Fragment>
+      );
+    } else {
+      content = this.renderEmpty();
+    }
+
+    return (
+      <EuiPage>
+        <EuiPageBody>
+          <EuiPageContent>
+            {this.renderConfirmModal()}
+            <EuiFlexGroup justifyContent="spaceBetween" alignItems="flexEnd">
+              <EuiFlexItem grow={false}>
+                <EuiTitle size="l">
+                  <h1>
+                    <FormattedMessage
+                      id="xpack.indexLifecycleMgmt.policyTable.sectionHeading"
+                      defaultMessage="Index lifecycle policy management"
+                    />
+                  </h1>
+                </EuiTitle>
+              </EuiFlexItem>
+              {policies.length ? (
+                <EuiFlexItem grow={false}>{this.renderCreatePolicyButton()}</EuiFlexItem>
+              ) : null}
+            </EuiFlexGroup>
+            <EuiSpacer size="s" />
+            <EuiText>
```

Comment:
this copy seems unnecessary, we don't display similar messages on other CRUD management interfaces